dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GeneralWorkflowIT.java

```diff
@@ -131,6 +131,93 @@ private Workflow manualRegisterAndPublish(WorkflowsApi workflowsApi, String work
         return workflow;
     }
 
+    /**
+     * This tests that smart refresh correctly refreshes the right versions based on some scenarios for GitHub
+     */
+    @Test
+    public void testSmartRefreshGitHub() {
+        commonSmartRefreshTest(SourceControl.GITHUB, "DockstoreTestUser2/hello-dockstore-workflow", "testBoth");
+    }
+
+    /**
+     * This tests that smart refresh correctly refreshes the right versions based on some scenarios for Gitlab
+     */
+    @Test
+    public void testSmartRefreshGitlab() {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/gh/dockstore/dockstore/pullRequest?prid=6348842)